proxy/handlers.go

```diff
@@ -294,28 +268,23 @@ func (p *Proxy) UserDashboard(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := p.authenticate(w, r, session); err != nil {
-		log.FromRequest(r).Error().Err(err).Msg("proxy: authenticate failed")
-		httpErr := &httputil.Error{Code: http.StatusUnauthorized, CanDebug: true}
-		httputil.ErrorResponse(w, r, httpErr)
+		p.sessionStore.ClearSession(w, r)
```

Comment:
This looks new.  Intentional?

Reply:
Yes but I'll address in FUP PR with more tests and context.

Reply:
Makes sense to me